src/persistence/Elsa.Persistence.EntityFrameworkCore/DbContexts/OracleContext.cs

```diff
@@ -12,5 +13,92 @@ public class OracleContext:ElsaContext
         public OracleContext(DbContextOptions options) : base(options)
         {
         }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+
+            modelBuilder.Entity<ActivityDefinitionEntity>(entity =>
+            {
+
+                entity.Property(x => x.Description)
+                    .HasColumnType("NCLOB");
+
+                entity.Property(x => x.DisplayName)
+                    .HasColumnType("NCLOB");
+
+                entity.Property(x => x.Name)
+                    .HasColumnType("NCLOB");
+
+
+
```

Comment:
Any chance I can get you to remove the excessive whitespace (throughout this file)? 😃

Reply:
sure ,no problem :)